docs/notes_20170421.md

```diff
@@ -0,0 +1,9 @@
+
+## Talk with Jonathan
+
+* We should look to add colors
+    - it can help us add more template tricks
+    - also use concepts like printable
+* enable functions (?)
```

Comment:
lol, notice that question mark? it meant even I dont know what I wrote, sorry :P